_migration-assistant/deploying-migration-assistant/configuration-options.md

```diff
@@ -97,11 +97,20 @@ The following sample CDK performs a live capture migration with C&R:
             "passwordFromSecretArn": <TARGET_CLUSTER_PASSWORD_SECRET>
         }
     },
-    "captureProxyServiceEnabled": true,
-    "captureProxyExtraArgs": "",
+
+    "// settingsForCaptureAndReplay": "Enable the following services for live traffic capture and replay",
     "trafficReplayerServiceEnabled": true,
-    "trafficReplayerExtraArgs": "--speedup-factor 2.0",
-    "targetClusterProxyServiceEnabled": true
+
+    "// help trafficReplayerExtraArgs": "Increase speedup factor in order replay requests at a faster rate to catch up",
+    "trafficReplayerExtraArgs": "--speedup-factor 1.5",
+
+    "// help capture/target proxy pt. 1 of 2": "captureProxyService and targetClusterProxyService require networking access to be configured in order to successfully deploy",
+    "// help capture/target proxy pt. 2 of 2": "consider deploying without first and enabling after ensuring cluster networking access on the migration console",
```

Comment:
@AndreKurait: can you verify that this revision retains the correct technical meaning?

```suggestion
    "// help capture/target proxy pt. 2 of 2": "Consider deploying without the proxy first. After verifying cluster network access in the migration console, you can enable the proxy.",
```